lib/osf-components/addon/components/banners/view-only-link/component.ts

```diff
@@ -19,6 +20,7 @@ export default class BannersViewOnlyLink extends Component {
 
     @action
     stopViewOnly() {
-        this.router.transitionTo('home', { queryParams: { view_only: '' } });
+        const destinationPage = this.router.urlFor('home', { queryParams: { view_only: '' } });
+        WindowLocation.assignLocation(destinationPage);
```

Comment:
Why do we use `window.location.assign()` instead of route transitions?

Reply:
This is so the app will reload. Since we turned `refreshModel` to false, the view only parameter will stay there if we do a route transition. Unless there is a different type of route transition that can refresh the app?